tests/IntegrationTest/Spoonacular/Recommendation/HomeTest.test.jsx

```diff
@@ -0,0 +1,173 @@
+import { describe, test, expect } from "vitest";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import HomeTest from "src/pages/Home/HomeTest";
+import { BrowserRouter } from "react-router-dom";
+import axios from "axios";
+
+vi.mock("axios");
+
+describe("Home Page", () => {
+  test("render successfully", async () => {
+    const { getByText } = render(
+      <BrowserRouter>
+        <HomeTest />
+      </BrowserRouter>
+    );
+    console.log(document.body.innerHTML);
+  });
+
+  test("fetch recipe data successfully", async () => {
+    const recipeMock = [
+      {
+        cuisines: [("Mediterranean", "Italian", "European")],
+        dairyFree: true,
+        diets: ["dairy free"],
+        dishTypes: [("lunch", "main course", "main dish", "dinner")],
+        glutenFree: false,
+        id: 654614,
+        image: "https://spoonacular.com/recipeImages/654614-556x370.jpg",
+        readyInMinutes: 45,
+        servings: 4,
+        title: "Pappa Al Pomodoro",
+        vegan: false,
+        vegetarian: false,
+        veryHealthy: true,
+        veryPopular: false,
+        extendedIngredients: [{ name: "olive oil" }, { name: "garlic cloves" }],
+      },
+      {},
+    ];
+
+    axios.get.mockResolvedValue({
+      data: recipeMock,
+    });
+
+    // api called?
+    await waitFor(() => expect(axios.get).toHaveBeenCalledTimes(1));
+  });
+
+  test("fetch random recipe successfully", async () => {
+    const recipeMock = [
+      {
+        cuisines: [("Mediterranean", "Italian", "European")],
+        dairyFree: true,
+        diets: ["dairy free"],
+        dishTypes: [("lunch", "main course", "main dish", "dinner")],
+        glutenFree: false,
+        id: 654614,
+        image: "https://spoonacular.com/recipeImages/654614-556x370.jpg",
+        readyInMinutes: 45,
+        servings: 4,
+        title: "Pappa Al Pomodoro",
+        vegan: false,
+        vegetarian: false,
+        veryHealthy: true,
+        veryPopular: false,
+        extendedIngredients: [{ name: "olive oil" }, { name: "garlic cloves" }],
+      },
+      {},
+    ];
+
+    axios.get.mockResolvedValue({
+      data: recipeMock,
+    });
+
+    // api called to fetch random recipe?
+    expect(axios.get).toHaveBeenNthCalledWith(
+      1,
+      `${env.API_URL}//spoonacular/randomRecipe`,
```

Comment:
Did you mean to have two / before the "spoonacular/"s?
I saw it in other lines too.  Maybe you could replace them?